modules/analyze_plugin_example/src/Plugin/Analyze/Example.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\analyze_plugin_example\Plugin\Analyze;
+
+use Drupal\analyze\AnalyzePluginBase;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Url;
+
+/**
+ * Analyze plugin to display Example data.
```

Comment:
Really good and comprehensive example :star:

Reply:
On enabling the example module, the plugin gives a couple errors. Whilst the data is all fake, I think being able to enable to see it in action might be handy:
```
Error: Call to undefined method Drupal\analyze_plugin_example\Plugin\Analyze\Example::doStuff()

InvalidArgumentException: The URI '/example/url' is invalid. You must use a valid URI scheme.
```

Reply:
I don't think that's how example modules work, and the documentation makes it quite clear it shouldn't be enabled (and it can only be done via drush in any case).

Reply:
Resolved in dce7d21939b0bae2da7cf80291dd7d1c051cee99